CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # nf-core/metaboigniter: Changelog
 
+## v1.0.0dev - 2021-05-08
```

Comment:
```suggestion
## v1.1.0 dev

_..nothing yet.._
```

Reply:
Can't apply this one myself sorry as coming from an organsational repo so don't have the permissions. But is the wrong version number and we don't put release date as it's not released yet 👍🏻